src/dc_receive_imf.rs

```diff
@@ -2150,6 +2148,23 @@ fn set_better_msg(mime_parser: &mut MimeMessage, better_msg: impl AsRef<str>) {
     }
 }
 
+/// Returns the last message referenced from `References` header if it is in the database.
```

Comment:
the doc says `References` header but the function seems to use `InReplyTo` header only.

so, not sure, if the things are on-purpose - or i misunderstand sth. :)

Reply:
Good catch!

Both work in the test as `get_parent_message` is used only on DC messages and it sets both `InReplyTo` and `References` to the same value, but I always thought about `References` when writing this.

It is a mistake to check `InReplyTo` in case of message with a quote, because  `InReplyTo` refers to the quoted message instead of the last message in the chat in this case.